bsip-0069.md

```diff
@@ -156,15 +156,19 @@ struct head_block_time_ge_time_predicate
 
 ```
 
+### Activation of new predicates and removal of max_predicate_opcode parameter:
+
+A hard-fork date shall be chosen for activation of these new predicates and shall be sufficiently future-dated to allow time for nodes to upgrade.  The outdated committee parameter `max_predicate_opcode` shall be removed, as it is not needed for safe upgrade (see next section for discussion).
```

Comment:
Technically we're unable to remove the `max_predicate_opcode` field since it affects serialization of certain operation. We can have a hard fork to no longer use the parameter. And we can check whether the old guarding code can be removed, primarily due to proposals.

Reply:
Perhaps "disabled" instead of "removed".

Reply:
Not sure I know the specifics enough as to why it can't be removed (I guess any historical transactions that updated the parameter would become invalid if it was removed?), but I updated to say "disable, and, if possible without impacting history, remove".  Hopefully this covers sufficient latitude for voting purposes.